uniffi_bindgen/src/bindings/swift/templates/wrapper.swift

```diff
@@ -2,7 +2,9 @@
 // Trust me, you don't want to mess with it!
 
 import Foundation
-import uniffi_{{ ci.namespace() }}
+{% if is_testing -%}
+import {{ config.module_name() }}
+{% endif -%}
```

Comment:
This is the real meat of making it work.

The rest is convincing ourselves that it does.